src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => (calculator.result += num),
+    subtract: (num) => (calculator.result -= num),
+    multiply: (num) => (calculator.result *= num),
+    divide: (num) => {
+      if (num === 0) {
+        return;
```

Comment:
The `divide` method should return a meaningful value or throw an error when attempting to divide by zero, instead of just returning undefined. Consider handling this case more explicitly, perhaps by returning an error message or throwing an exception.